source/installation/source.rst

```diff
@@ -5,21 +5,20 @@ Source Code
 
 :audience:`administrators, developers`
 
-TANGO is available for download as source code or as pre-compiled binaries.
+Tango Controls for C++ and Java is available for download as source code or as pre-compiled binaries.
 
-Always look at the Patches page (available just below) when you download a Tango release.
-If some patches are available for your release, please, get and apply them.
-
-The latest source code release is `9.2.5a <https://sourceforge.net/projects/tango-cs/files/tango-9.2.5a.tar.gz/download>`_  for C++ and Java
-
-   * `Readme - TANGO changes <http://ftp.esrf.fr/pub/cs/tango/README.9.2.5a.txt>`_
-   * `MD5 <ftp.esrf.fr/pub/cs/tango/tango-9.2.5a.tar.gz.md5>`_
+`The latest source code release is 9.3.3-rc1 <https://github.com/tango-controls/TangoSourceDistribution/releases/tag/9.3.3-rc1>`_.
 
+   * `Tango 9.3.3 C++ library changelog <https://github.com/tango-controls/cppTango/releases/tag/9.3.3>`_
```

Comment:
I would use "release notes" instead of "changelog" in this case...

Reply:
Done